apps/console/src/features/identity-providers/components/forms/helpers/form-fields-helper.tsx

```diff
@@ -281,34 +281,35 @@ export const getTableField = (eachProp: CommonPluggableComponentPropertyInterfac
                     }
                 } }
             />
-            { eachProp?.value.split(" ").map(scope => {
-                return (
-                    <Grid key={ scope } verticalAlign="middle">
-                        <Grid.Row columns={ 2 }>
-                            <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 2 }>
-                                <GenericIcon
-                                    icon={ propertyMetadata.properties[scope].icon }
-                                    size="micro"
-                                    square
-                                    transparent
-                                    inline
-                                    className="left-icon"
-                                    verticalAlign="middle"
-                                    spaced="right"
-                                />
-                                <label data-testid={ `${ testId }-authorize-label` }>
-                                    { scope }
-                                </label>
-                            </Grid.Column>
-                            <Grid.Column mobile={ 8 } tablet={ 8 } computer={ 14 }>
-                                <label data-testid={ `${ testId }-authorize-label` }>
-                                    { propertyMetadata.properties[scope].description }
-                                </label>
-                            </Grid.Column>
-                        </Grid.Row>
-                    </Grid>
-                );
-            }) }
+            <Grid>
+                { eachProp?.value.split(" ").map((scope,index) => (
+                    <Grid.Row columns={ 3 } key={ index }>
+                        <Grid.Column mobile={ 1 } tablet={ 1 } computer={ 1 }>
+                            <GenericIcon
+                                icon={ propertyMetadata.properties[scope].icon }
+                                size="micro"
+                                square
+                                transparent
+                                inline
+                                className="left-icon"
+                                verticalAlign="middle"
+                                spaced="left"
+                            />
+                        </Grid.Column>
+                        <Grid.Column mobile={ 2 } tablet={ 2 } computer={ 2 }>
+                            <Label data-testid={ `${ testId }-authorize-label` } className="scope-name">
```

Comment:
Why do we need a `Label` here. IMO, we can get away with just a `<div>` tag here.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1952/commits/16dcc25d2ba3e4dc779005ef740ac930cd671ea5 by adding a div tag